src/Hazelcast.Net/DistributedObjects/IHCollection.cs

```diff
@@ -18,18 +18,13 @@
 
 namespace Hazelcast.DistributedObjects
 {
-    /// <summary>Concurrent, distributed, partitioned, listenable collection.</summary>
-    /// <remarks>Concurrent, distributed, partitioned, listenable collection.</remarks>
+    /// <summary>
+    /// Concurrent, distributed, partitioned, listenable collection.
+    /// </summary>
+    /// <typeparam name="T">item</typeparam>
```

Comment:
"The type of the items in the collection"

Reply:
ok